client/src/components/Navigation.tsx

```diff
@@ -1,24 +1,52 @@
-import { ReactElement } from "react";
-import {Nav, Navbar, NavLink} from 'react-bootstrap';
-import {Link} from "react-router-dom";
-import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import {AppBar, Typography, Toolbar, Button} from '@mui/material';
+import { Link } from 'react-router-dom';
+import { styled } from '@mui/system';
+import { ReactElement } from 'react';
```

Comment:
You could merge this line with line 1 to have
import React, { ReactElement } from 'react';
instead of two separate lines